src/sentry/features/__init__.py

```diff
@@ -107,6 +107,7 @@
 default_manager.add("organizations:performance-tag-explorer", OrganizationFeature, True)  # NOQA
 default_manager.add("organizations:performance-view", OrganizationFeature)  # NOQA
 default_manager.add("organizations:related-events", OrganizationFeature)  # NOQA
+default_manager.add("organizations:release-adoption-chart", OrganizationFeature)  # NOQA
```

Comment:
do we have to put true for this so that it is managed by flagr?

Reply:
True